registry/registry.go

```diff
@@ -219,6 +220,10 @@ func ContinueOnError(err error) bool {
 		return shouldV2Fallback(v)
 	case *client.UnexpectedHTTPResponseError:
 		return true
+	case error:
+		if val := strings.Contains(err.Error(), strings.ToLower(syscall.ENOSPC.Error())); val {
```

Comment:
@RichardScothern @cpuguy83 is right `ENOSPC.Error()` string returned is a part of the larger string returned from `chrootarchive.invokeUnpack()` therefore we need to do sub-string search instead of comparing the error code, unfortunately. But going forward, it may better to return standardize error codes return values to caller. Thanks.